src/Symfony/Component/Security/Tests/Core/Encoder/BCryptPasswordEncoderTest.php

```diff
@@ -45,9 +45,6 @@ public function testCostInRange()
         }
     }
 
-    /**
-     * @requires PHP 5.3.7
```

Comment:
By the way, that's the reason why tests on Appveyor are failing now (see #17073 for the fix).